src/main/java/com/minecolonies/coremod/commands/colonycommands/MakeNotAutoDeletableCommand.java

```diff
@@ -1,23 +1,29 @@
 package com.minecolonies.coremod.commands.colonycommands;
 
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Map'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)